pages/1.11/release-notes/1.11.4/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.4
+title: Release Notes for 1.11.4
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.4
+---
+
+DC/OS 1.11.4 was released on July 26, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.4/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.4 includes the following:
+
+- Apache Mesos 1.5.2-dev [change log](https://github.com/mesosphere/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
+- Marathon 1.6.496 [change log](https://github.com/dcos/dcos/blob/1.11.4/packages/marathon/buildinfo.json).
```

Comment:
`marathon-1.6.535`

Reply:
Please change the URL to https://github.com/mesosphere/marathon/releases/tag/v1.6.535